ui/ui-expenses/src/screens/prices/prices.reactions.js

```diff
@@ -0,0 +1,22 @@
+export const send = ((action, store, { http }) => {
```

Comment:
According to what I have understand about the user flow, the button send on the header, visible only on the screen `list`, send the list of the pending expenses.
When you create a new expense, you add it to the list but don't send it directly to the accountant (so the list need to be persisted)

Reply:
You can add a reaction `add` that add the created expense to the list of pending expenses, and the button `valider` (style coming later) in the screen if you are in the mood 😄